script/v2/keeper-actions/RebalanceScWethV2.s.sol

```diff
@@ -108,6 +110,9 @@ contract RebalanceScWethV2 is Script, scWETHv2Helper {
 
         _initializeAdapterSettings();
 
+        // if the vault has float less than minimumFloat amount, withdraw it to the vault
+        _updateVaultFloat();
+
         uint256 investAmount = _calcInvestAmount();
```

Comment:
in the situation where we need to withdraw to restore float, the investAmount is always going to be > min required float, right? consider adding a check to set the investAmount to 0 in this case so we don't end up investing during rebalance what if  we were withdrawing in the previous transaction

Reply:
I am not sure you I understand your question correctly. In a situation where there are no funds/float to invest but we need to reinvest profits (increase the LTVs), investAmount will be 0, so that in that case investAmount will be less than minFloatAmoun.

Reply:
what i meant to say is this:
1. float is under min required
2. we call withdraw with the missing amount + 5% more
3. now float is > min required
4. we calculate the `investAmount` as float - min required, which in this case equals whatever is left from that 5% more we initially withdrew 
5. now we have `investAmount` > 0 so we will basically invest here ~5% we withdrew in step 2

Reply:
Oh yes good point. But we don't need a check for that here, since we already have a minimumInvestAmount check. The rebalance method won't be called if investAmount is less than that.

<img width="632" alt="Screenshot 2023-11-01 at 11 03 31 AM" src="https://github.com/lindy-labs/sandclock-contracts/assets/47485188/e89da452-03da-4f98-9aa5-896af95a6d69">